src/Symfony/Component/PropertyInfo/Extractor/PhpDocExtractor.php

```diff
@@ -68,7 +68,7 @@ public function __construct(DocBlockFactoryInterface $docBlockFactory = null, ar
     /**
      * {@inheritdoc}
      */
-    public function getShortDescription($class, $property, array $context = [])
+    public function getShortDescription(string $class, string $property, array $context = [])
```

Comment:
Return types should aslo be added, isn't it?

Reply:
Got it, we cannot yet: 30323.

Reply:
No, we can't add return types as they are not covered by [Parameter Type Widening](https://wiki.php.net/rfc/parameter-no-type-variance). See also @xabbuh's comment here: https://github.com/symfony/symfony/pull/32202#discussion_r298078537

Reply:
But we could add return types to private methods, though.